src/plugins/core/pivot.ts

```diff
@@ -252,6 +241,14 @@ export class PivotCorePlugin extends CorePlugin<CoreState> implements CoreState
     });
   }
 
+  private getPivotCore(pivotId: UID): Pivot {
+    const pivot = this.pivots[pivotId];
+    if (!pivot) {
+      throw new Error(`Pivot with id ${pivotId} not found`);
```

Comment:
we should we do with this error? review the revision log and fix it ?

Reply:
The only 
It should not happen, if it happens, it's a bug and we will have to find the cause to fix it, that could be revision log, but not only